GUI/Controls/ModInfoTabs/Metadata.cs

```diff
@@ -72,10 +72,10 @@ public void UpdateModInfo(GUIMod gui_module)
                     ReplacementTextBox.Text = module.replaced_by.ToString();
                 }
 
-                if (gui_module.DownloadCount == null)
+                if (gui_module.DownloadCount == null or gui_module.DownloadCount == 0)
                 {
                     DownloadCountTextBox.Visible = false;
-                    MetadataTable.LayoutSettings.RowStyles[0].Height = 0;
+                    MetadataTable.LayoutSettings.RowStyles[0].Height = 3;
```

Comment:
I'm pretty sure the value being 0 when setting visble to false was correct, but you need the index of the DownloadCount row, which i believe would be 7 but do not quote me on that

Reply:
The row is 3, and it goes between the brackets; the height goes after the equals sign:

```suggestion
                    MetadataTable.LayoutSettings.RowStyles[3].Height = 0;
```